modules/javafx.base/src/main/java/javafx/beans/value/ObservableValue.java

```diff
@@ -251,4 +252,63 @@ default ObservableValue<T> orElse(T constant) {
     default <U> ObservableValue<U> flatMap(Function<? super T, ? extends ObservableValue<? extends U>> mapper) {
         return new FlatMappedBinding<>(this, mapper);
     }
+
+    /**
+     * Returns an {@code ObservableValue} that holds this value whenever the given
+     * condition evaluates to {@code true}, otherwise holds the last value when
+     * {@code condition} became {@code false}. The value is updated whenever this
```

Comment:
What I'm trying to communicate here is that the last value held by the parent will be the value held by this observable at the time the condition becomes false.  Perhaps more accurately:

> Returns an `ObservableValue` that holds this value and is updated only when the given condition evaluates to `true`.

This also communicates that it will initially copy the value, and then keeps copying it for as long as condition is `true`. It also implies (with "updated") that the value will remain at the last copied value for the duration of the condition being `false`.